packages/logger/src/LoggerFactory.ts

```diff
@@ -0,0 +1,16 @@
+import { ConsolaStructuredLogger } from "./ConsolaStructuredLogger";
+import { GCPStructuredLog } from "./structuredLog/GCPStructuredLog";
+import { LoggerOptions } from "./interfaces/LoggerOptions";
+
+type LoggerType = "consola-structured"; // | "winston" | "pino";
+
+export class LoggerFactory {
+  static create(type: LoggerType, options: LoggerOptions = {}) {
+    switch (type) {
```

Comment:
Can we assume that we have an opinion about the default logger and use it if someone won't pass the type?